test/specs/manifestBuilder.e2e.ts

```diff
@@ -0,0 +1,225 @@
+  //   );
+  //   // TODO: Find a way to use SFDX: Generate Manifest File
+
+  //   // // Set the name of the new manifest file
+  //   // await inputBox.setText('manifest');
+  //   // await inputBox.confirm();
+
+  //   // // Wait for the command to execute
+  //   // const workbench = await browser.getWorkbench();
+  //   // await utilities.waitForNotificationToGoAway(
+  //   //   workbench,
+  //   //   'Running SFDX: Generate Manifest File',
+  //   //   fiveMinutes
+  //   // );
+
+  //   // // Look for the success notification that appears which says, "SFDX: Generate Manifest File successfully ran".
+  //   // const successNotificationWasFound = await utilities.notificationIsPresent(
+  //   //   workbench,
+  //   //   'SFDX: Generate Manifest File successfully ran'
+  //   // );
+  //   // expect(successNotificationWasFound).toBe(true);
+
+  //   // // Verify manifest file was created
+  //   // const editorView = workbench.getEditorView();
+  //   // const activeTab = await editorView.getActiveTab();
+  //   // const title = await activeTab?.getTitle();
+  //   // expect(title).toBe('manifest.xml');
+
+  //   // // Verify Output tab
+  //   // const outputPanelText = await utilities.attemptToFindOutputPanelText(
+  //   //   'Salesforce CLI',
+  //   //   'Starting SFDX: Generate Manifest File',
+  //   //   10
+  //   // );
+  //   // expect(outputPanelText).not.toBeUndefined();
+  //   // expect(outputPanelText).toContain('ended SFDX: Generate Manifest File');
+  // });
+
+  step('SFDX: Deploy Source in Manifest to Org', async () => {
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - SFDX: Deploy Source in Manifest to Org`
+    );
+    // Using the Command palette, run SFDX: Deploy Source in Manifest to Org
+    const workbench = await browser.getWorkbench();
+    // Clear output before running the command
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'View: Clear Output',
+      1
+    );
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'SFDX: Deploy Source in Manifest to Org',
+      1
+    );
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(
+      workbench,
+      'Running SFDX: Deploy Source to Org',
+      fiveMinutes
+    );
+
+    // Look for the success notification that appears which says, "SFDX: Deploy Source to Org successfully ran".
+    const successNotificationWasFound = await utilities.notificationIsPresent(
+      workbench,
+      'SFDX: Deploy Source to Org successfully ran'
+    );
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify Output tab
+    const outputPanelText = await utilities.attemptToFindOutputPanelText(
+      'Salesforce CLI',
+      'Starting SFDX: Deploy Source to Org',
+      10
+    );
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('=== Deployed Source');
```

Comment:
I think we need to be careful here, because we don't control that output that's generated.  Maybe change lines 151-162 with:
```
    expect(outputPanelText).not.toBeUndefined();
    expect(outputPanelText).toContain('Created  Customer__c  CustomObject');
    expect(outputPanelText).toContain('Created  Product__c   CustomObject');
    expect(outputPanelText).toContain('ended SFDX: Deploy Source to Org');
```

...or:
```
    expect(outputPanelText).not.toBeUndefined();
    expect(outputPanelText).toContain('Created  Customer__c  CustomObject  force-app/main/default/objects/Customer__c/Customer__c.object-meta.xml');
    expect(outputPanelText).toContain('Created  Product__c   CustomObject  force-app/main/default/objects/Product__c/Product__c.object-meta.xml');
    expect(outputPanelText).toContain('ended SFDX: Deploy Source to Org');
```


What are your thoughts?

Reply:
I think it's ok to remove all the titles but I'd keep "Deployed source". Also I wouldn't specify created, only the rest of the substring (x__c  CustomObject  force-app/main/default/objects/x__c/x__c.object-meta.xml)